### DISCUSSION TITLE
Using HoneybadgerJS vs using HoneybadgerVue or Honeybadger React

### DISCUSSION THREAD

**Discussion by shawn-m-tech** [honeybadger-io/honeybadger-js#790] [Q&A] [answered]
I'll start off by saying I absolutely love your mascot!
![image](https://user-images.githubusercontent.com/45756873/168181598-d9c3589f-8d39-4245-bba7-c3ddbde56939.png)

Next, excuse my ignorance, but I'm curious about the benefits of using Honeybadger Vue instead of just using Honeybadger JS for apps that are primarily built on Vue (Maybe 70% Vue 30% Rails ERB).

Thanks so much

P.S. What is your Mascot's name?

**Comment by subzero10** [comment, marked as answer]
Hello @shawn-p-m,

Thank you for your kind words! Unfortunately, our mascot does not have a name yet, but we should come up with one soon! Feel free to share your thoughts :)

To answer your question:
Honeybadger Vue uses the core Honeybadger Js client. It registers an error handler for vue errors which allows reporting errors automatically to Honeybadger. It also sets up a `$honeybadger` variable available to all components for easier manual reporting.
You can install `@honeybadger-io/vue` for `vue 2.x` or `@honeybadger-io/vue@next` for `vue 3.x`.

Does this help?

**Reply by shawn-m-tech** [reply]
Yes, that does help. Thank you!

No idea what the mascot should be named haha. Something awesome though for sure 😝.  "Damian"?

**Reply by subzero10** [reply]
Hehe, "Damian" is not bad! 😄
